pkg/storaged/lvol-tabs.jsx

```diff
@@ -50,11 +50,12 @@ function lvol_rename(lvol) {
     });
 }
 
-function lvol_and_fsys_resize(client, lvol, size, offline, passphrase) {
+function lvol_and_fsys_resize(client, lvol, targetSize, offline, passphrase) {
     var block, crypto, fsys;
     var crypto_overhead;
     var vdo;
     var orig_size = lvol.Size;
+    var size = parseInt(targetSize);
```

Comment:
Do you know where the double comes from?  The size slider shouldn't produce any...

Reply:
And, yeah, parseInt is a weird way to turn a double into an int, no?

Reply:
It's coming from the text input.

Reply:
The docs say parseInt takes a string as an argument, but passing double works as well because JS.
Tried:

parseInt(1234.5)
1234

Reply:
I see, then we should call Math.round() also in `change_text` of SizeSliderElement.

Reply:
And we need a test as well. :-)

Reply:
> The docs say parseInt takes a string as an argument, but passing double works as well because JS.

Yes, it works, but it's still weird, imo.  Math.round() or Math.floor() is much cleaner.

Reply:
Right, just realized it as well.

Actually the user experience of the input field is pretty horrible. We change the input inline while the user types, therefore the user can't just use backspace and insert some new value when the input field is a double number at least.
I am even considering if we should disable that input component (make it readonly). Because we clearly cannot accept every value the user will insert.